app/services/typeahead_service.rb

```diff
@@ -125,7 +79,8 @@ def search_profiles(search_string, limit = 5)
   def search_notes(search_string, limit = 5)
```

Comment:
Maybe we should add to the comment to say "and package up as a TagResult" for clarity?

Reply:
yeah, good point!